pkg/csi/service/wcpguest/controller.go

```diff
@@ -716,6 +717,12 @@ func controllerUnpublishForBlockVolume(ctx context.Context, req *csi.ControllerU
 	}
 	var err error
 	if err := c.vmOperatorClient.Get(ctx, vmKey, virtualMachine); err != nil {
+		if err != nil {
```

Comment:
This line is redundant since we're already checking on line 719

Reply:
My bad! fixed